src/medCore/gui/viewContainers/medViewContainer.cpp

```diff
@@ -926,6 +942,58 @@ QString medViewContainer::saveScene()
     return "";
 }
 
+void medViewContainer::addMetadataToQDomElement(medAbstractData* data, QDomElement patientInfo, QString metadata)
+{
+    if (data->hasMetaData(metadata) && data->metadata(metadata) != "")
+    {
+        patientInfo.setAttribute(metadata, data->metadata(metadata));
+    }
+}
+
+void medViewContainer::addMetadataToMedAbstractData(medAbstractData* data, QString value, QString metadata)
+{
+    if (value != "")
+    {
+        if (data->hasMetaData(metadata))
+        {
+            data->setMetaData(metadata, value);
+        }
+        else
+        {
+            data->addMetaData(metadata, value);
+        }
+    }
+}
+
+bool medViewContainer::isRemoveAndAddMedAbstractDataNeeded(medAbstractData* data, QString patientName, QString studyDescription)
```

Comment:
What does it do?
I think you should change the method's name, it's not quite clear.